benchmarks/benchmarks.py

```diff
@@ -0,0 +1,84 @@
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the asv docs for more information.
+import numpy as np
+import scipy.stats as st
+
+
+class Hist:
+	def time_histogram(self):
```

Comment:
These look like tabs not spaces :) 

Can you run black over this file for consistent formatting of all .py files?